src/screens/FeedScreen/Header.js

```diff
@@ -444,7 +444,7 @@ const styles = StyleSheet.create({
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 16.94,
-    color: colors.black
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been updated from `colors.black` to `COLORS.black`. This change is consistent with the update in the import statement.
```diff
- color: colors.black
+ color: COLORS.black
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->